src/org/usfirst/frc/team2399/robot/subsystems/DriveTrain.java

```diff
@@ -178,18 +184,108 @@ public double returnRightDistanceError(){
 	 * the P to the current P + the increment/decrement constant
 	 */
 	
-	public void incrementDistanceConstant(CANTalon currentTalon){
-		double currentPConstant = currentTalon.getP();
-		currentTalon.setP(currentPConstant += RobotMap.DISTANCE_INCREMENT);
+	public void incrementRightDistancePConstant(){
+		double currentPConstant = rightFrontTalon.getP();
+		rightFrontTalon.setP(currentPConstant += RobotMap.DRIVE_DISTANCE_INCREMENT);
+	}
+	
+	public void incrementLeftDistancePConstant(){
+		double currentPConstant = leftFrontTalon.getP();
+		leftFrontTalon.setP(currentPConstant += RobotMap.DRIVE_DISTANCE_INCREMENT);
+	}
+	
+	public void decrementRightDistancePConstant(){
+		double currentPConstant = rightFrontTalon.getP();
+		rightFrontTalon.setP(currentPConstant -= RobotMap.DRIVE_DISTANCE_DECREMENT);
+	}
+	
+	public void decrementLeftDistancePConstant(){
+		double currentPConstant = leftFrontTalon.getP();
+		leftFrontTalon.setP(currentPConstant -= RobotMap.DRIVE_DISTANCE_DECREMENT);
+	}
+	
+	public double returnRightDistanceConstant(){
+		return rightFrontTalon.getP();
+	}
+	
+	public double returnLeftDistanceConstant(){
+		return leftFrontTalon.getP();
+	}
+	
+	public double getCurrentAngle()
+	{
+		return Navx.getYaw();
```

Comment:
Update: See http://www.kauailabs.com/public_files/navx-mxp/apidocs/java/ -> AHRS -> getYaw; return value is in range (-180.0, 180.0).